packages/studio-ui/src/utils/loadComponents.tsx

```diff
@@ -0,0 +1,27 @@
+import useStudioStore from "../store/useStudioStore";
+import getFunctionComponent from "./getFunctionComponent";
+import dynamicImportFromBrowser from "./dynamicImportFromBrowser";
+
+/**
+ * Load all of the user's components into the store.
+ */
+export default function loadComponents(): Promise<void>[] {
+  const { UUIDToFileMetadata, setImportedComponent, UUIDToImportedComponent } =
+    useStudioStore.getState().fileMetadatas;
+
+  const componentImportPromises = Object.values(UUIDToFileMetadata).map(
+    async (fileMetadata) => {
+      if (UUIDToImportedComponent.hasOwnProperty(fileMetadata.metadataUUID)) {
```

Comment:
also for my edification, would something like `if (fileMetadata.metadataUUID in UUIDToImportedComponent)` be more readable, or is there a functional difference for this case?

Reply:
its OK, found an article: https://masteringjs.io/tutorials/fundamentals/hasownproperty

Reply:
oh yeah distinction is small and 99% of the time unimportant